src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -77,78 +78,51 @@ class MilitaryInformationContent extends React.Component {
     }
 
     return (
-      <table className="militaryInformation" data-field-name="serviceHistory">
-        <thead>
-          <tr>
-            <th>
-              <h3>Period of service</h3>
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          {serviceHistory.map((service, index) => (
-            <tr key={index}>
-              <td>
-                <h4 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--bold">
-                  {service.branchOfService}
-                </h4>
-              </td>
-              <td>
-                {moment(service.beginDate).format('MMM D, YYYY')} &ndash;{' '}
-                {moment(service.endDate).format('MMM D, YYYY')}
-              </td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
-    );
-  };
-
-  render() {
-    return (
-      <div className="vads-u-margin-bottom--2">
-        <LoadingSection
-          isLoading={!this.props.militaryInformation}
-          message="Loading military information..."
-          render={this.renderContent}
+      <>
+        <ProfileInfoTable
+          data={transformServiceHistoryForTable(serviceHistory)}
+          title="Period of service"
+          fieldName="serviceHistory"
         />
-        <AdditionalInfo
-          triggerText="What if my military service information doesn’t look right?"
-          onClick={() => {
-            recordEvent({
-              event: 'profile-navigation',
-              'profile-action': 'view-link',
-              'profile-section': 'update-military-information',
-            });
-          }}
-        >
-          <p>
-            Some Veterans have reported seeing military service information in
-            their VA.gov profiles that doesn’t seem right. When this happens,
-            it’s because there’s an error in the information we’re pulling into
-            VA.gov from the Defense Enrollment Eligibility Reporting System
-            (DEERS).
-          </p>
-          <p>
-            If the military service information in your profile doesn’t look
-            right, please call the Defense Manpower Data Center (DMDC). They’ll
-            work with you to update your information in DEERS.
-          </p>
-          <p>
-            To reach the DMDC, call{' '}
-            <a
-              href="tel:1-800-538-9552"
-              aria-label="8 0 0. 5 3 8. 9 5 5 2."
-              title="Dial the telephone number 800-538-9552"
-              className="no-wrap"
-            >
-              1-800-538-9552
-            </a>
-            , Monday through Friday (except federal holidays), 8:00 a.m. to 8:00
-            p.m. ET. If you have hearing loss, call TTY: 1-866-363-2883.
-          </p>
-        </AdditionalInfo>
-      </div>
+        <div className="vads-u-margin-y--4">
```

Comment:
the rendering logic has changed a bit so that this `AdditionalInfo` component is only rendered if there is actually service history to show.